src/ol/view.js

```diff
@@ -256,10 +256,17 @@ ol.View.prototype.getCenter = function() {
 
 
 /**
+ * @param {Array.<number>=} opt_hints Destination array.
  * @return {Array.<number>} Hint.
  */
-ol.View.prototype.getHints = function() {
-  return this.hints_.slice();
+ol.View.prototype.getHints = function(opt_hints) {
```

Comment:
I'm not opposed to this.  But do you see any problem with just returning `this.hints_`?